src/injected/content/clipboard.js

```diff
@@ -1,13 +1,8 @@
 import { isFirefox } from '#/common/ua';
 import { sendCmd } from '../utils';
-import { addEventListener, logging } from '../utils/helpers';
+import { logging } from '../utils/helpers';
 import bridge from './bridge';
 
-const { execCommand } = Document.prototype;
-const { setData } = DataTransfer.prototype;
-const getClipboardData = Object.getOwnPropertyDescriptor(ClipboardEvent.prototype, 'clipboardData').get;
-const { preventDefault, removeEventListener, stopImmediatePropagation } = EventTarget.prototype;
```

Comment:
The real problem is, `preventDefault` and `stopImmediatePropagation` comes from `Event.prototype`. I corrected it locally and it worked in Firefox!

Reply:
Sorry, there is one more thing. The getter `clipboardData` should also be kept.